flags/flags.go

```diff
@@ -52,38 +57,102 @@ unset, default is P-256 for EC keys and Ed25519 for OKP keys.
 
 		**Ed25519**
 		:  Ed25519 Curve`,
-}
+	}
 
-// Subtle is the flag required for delicate operations.
-var Subtle = cli.BoolFlag{
-	Name: "subtle",
-}
+	// Subtle is the flag required for delicate operations.
+	Subtle = cli.BoolFlag{
+		Name: "subtle",
+	}
 
-// Insecure is the flag required on insecure operations
-var Insecure = cli.BoolFlag{
-	Name: "insecure",
-}
+	// Insecure is the flag required on insecure operations
+	Insecure = cli.BoolFlag{
+		Name: "insecure",
+	}
 
-// Force is a cli.Flag used to overwrite files.
-var Force = cli.BoolFlag{
-	Name:  "f,force",
-	Usage: "Force the overwrite of files without asking.",
-}
+	// Force is a cli.Flag used to overwrite files.
+	Force = cli.BoolFlag{
+		Name:  "f,force",
+		Usage: "Force the overwrite of files without asking.",
+	}
 
-// PasswordFile is a cli.Flag used to pass a file to encrypt or decrypt a
-// private key.
-var PasswordFile = cli.StringFlag{
-	Name:  "password-file",
-	Usage: `The path to the <file> containing the password to encrypt or decrypt the private key.`,
-}
+	// PasswordFile is a cli.Flag used to pass a file to encrypt or decrypt a
+	// private key.
+	PasswordFile = cli.StringFlag{
+		Name:  "password-file",
+		Usage: `The path to the <file> containing the password to encrypt or decrypt the private key.`,
+	}
 
-// NoPassword is a cli.Flag used to avoid using a password to encrypt private
-// keys.
-var NoPassword = cli.BoolFlag{
-	Name: "no-password",
-	Usage: `Do not ask for a password to encrypt a private key. Sensitive key material will
+	// NoPassword is a cli.Flag used to avoid using a password to encrypt private
+	// keys.
+	NoPassword = cli.BoolFlag{
+		Name: "no-password",
+		Usage: `Do not ask for a password to encrypt a private key. Sensitive key material will
 be written to disk unencrypted. This is not recommended. Requires **--insecure** flag.`,
-}
+	}
+
```

Comment:
I assume we should be switching to using these global flags, rather than the private ones defined in /ca and /certificate and /ssh.

Reply:
I've switched to using the global flags where possible